tests/test_initialization.py

```diff
@@ -4477,3 +4477,67 @@ def test_key_mapping_save_old_load_new_vblora(self, old_model, new_model, tmp_pa
     def test_key_mapping_save_new_load_old_vblora(self, old_model, new_model, tmp_path):
         # save the new model, load it into the old model, should work without issues (forwards compatibility)
         self.check_vblora_load_no_warning(new_model, old_model, tmp_path)
+
+
+class TestKasaInitialization:
```

Comment:
Could you please move the test functions to the bottom of `class TestLoraInitialization`? This is because KaSA is a LoRA variant, and it also means we can reuse the existing `get_model` method.